coopersystem-fsd/pcc
# Adjusted the content layout

## Description

- Removed border between course details and topics;
- Removed font weight bold from course details and topics;
- Aligned the content title to the top when .is-layout-flex class is set.

## Steps to test

1. See a course page;
2. See "Sustain our work by joining today:" at https://platform.coop/about/pcc-solidarity-colaboratory/

## Additional information
